kfs-core/src/main/resources/edu/arizona/kfs/db/changelog/changesets/db.changelog-UAF-37.xml

```diff
@@ -0,0 +1,36 @@
+<?xml version="1.0" encoding="UTF-8"?>
+ 
+<databaseChangeLog
+    xmlns="http://www.liquibase.org/xml/ns/dbchangelog" 
+    xmlns:xsi="http://www.w3.org/2001/XMLSchema-instance" 
+    xsi:schemaLocation="http://www.liquibase.org/xml/ns/dbchangelog 
+        http://www.liquibase.org/xml/ns/dbchangelog/dbchangelog-3.1.xsd">
+		
+	<changeSet id="UAF-37" author="Julie Wingate">
+		<preConditions onError="MARK_RAN" onFail="MARK_RAN">
+			<sqlCheck expectedResult="0">
+				SELECT COUNT(*) FROM KRCR_PARM_T
+					WHERE parm_nm='ALLOW_MANY_TO_MANY_TRANSFERS_IND'
+			</sqlCheck>
+		</preConditions>
+		<comment>
+			UAF-37 Block a Many to Many transfer within the TF and YETF documents
+		</comment>
+		<insert tableName="KRCR_PARM_T">
+            <column name="NMSPC_CD" value="KFS-FP"/>
```

Comment:
Will you fix the indentation on the column and closing insert tag? Then this will be easier to read.